CMakeLists.txt

```diff
@@ -70,6 +71,7 @@ if(MSVC)
 
 else()
 
+  set(libsuffix .so)
```

Comment:
```suggestion
  # We can not use CMAKE_SHARED_LIBRARY_SUFFIX as on macos it uses the standard dylib rather than the .so
  set(libsuffix .so)
```
assuming that this is the correct reason.

Reply:
right, thanks!